requirements.txt

```diff
@@ -1,9 +1,4 @@
-pdfrw==0.4
-jupyterlab==0.35.4
```

Comment:
I think @georgehelman uses`jupyterlab` in his workflow. Maybe we could make a separate requirements file for local development stuff

Reply:
Yeah, I use it too, but I don't think it needs to be apart of our CI pipeline. It's easy enough to pip install manually if needed.